api/libproject/devstart.rb

```diff
@@ -634,6 +634,50 @@ def run_local_all_migrations()
   :fn => ->() { run_local_all_migrations() }
 })
 
+def rollback_db(cmd_name, *args)
+  puts "in rollback_db. cmd_name: #{cmd_name} args: #{args}"
+  ensure_docker_sync
+  op = WbOptionsParser.new(cmd_name, args)
+  op.add_typed_option(
+      "--count=[count]",
+      Integer,
+      ->(opts, c) { opts.count = c},
+      "Count of changesets to roll back from end.")
+
+  op.add_typed_option(
+      "--tag [tag]",
+      String,
+      ->(opts, t) { opts.tag = t},
+      "Liquibase changeset tag to roll back to.")
+  op.add_validator ->(opts) {
```

Comment:
This was a pain to write, and Zeus help me if I want another option. The idea is that exactly one is provided.

Also, I want a general liquibase command. I'll do that tomorrow just for fun.